[1.7.10] Crash when analyze flower Delphinium in FieldKit

```SimpleChannelHandlerWrapper exception
java.lang.RuntimeException: java.lang.NullPointerException
        at binnie.core.network.BinniePacketHandler.onMessage(BinniePacketHandler.java:31) ~[BinniePacketHandler.class:?]
        at binnie.core.network.BinniePacketHandler.onMessage(BinniePacketHandler.java:9) ~[BinniePacketHandler.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleChannelHandlerWrapper.channelRead0(SimpleChannelHandlerWrapper.java:37) ~[SimpleChannelHandlerWrapper.class:1.7.10-1614.58]
        at cpw.mods.fml.common.network.simpleimpl.SimpleChannelHandlerWrapper.channelRead0(SimpleChannelHandlerWrapper.java:17) ~[SimpleChannelHandlerWrapper.class:1.7.10-1614.58]
        at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:98) ~[SimpleChannelInboundHandler.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337) [DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323) [DefaultChannelHandlerContext.class:?]
        at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) [MessageToMessageDecoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:111) [MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337) [DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323) [DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:785) [DefaultChannelPipeline.class:?]
        at io.netty.channel.embedded.EmbeddedChannel.writeInbound(EmbeddedChannel.java:169) [EmbeddedChannel.class:?]
        at cpw.mods.fml.common.network.internal.FMLProxyPacket.func_148833_a(FMLProxyPacket.java:77) [FMLProxyPacket.class:1.7.10-1614.58]
        at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:245) [ej.class:?]
        at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:181) [nc.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:1023) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:432) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:841) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:693) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_131]
Caused by: java.lang.NullPointerException
        at net.minecraft.item.ItemStack.func_77969_a(Unknown Source) ~[add.class:?]
        at binnie.botany.genetics.FlowerHelper.getConversion(FlowerHelper.java:249) ~[FlowerHelper.class:?]
        at binnie.core.genetics.FlowerBreedingSystem.getConversion(FlowerBreedingSystem.java:79) ~[FlowerBreedingSystem.class:?]
        at binnie.core.genetics.ManagerGenetics.getConversionSystem(ManagerGenetics.java:125) ~[ManagerGenetics.class:?]
        at binnie.core.genetics.ManagerGenetics.getConversionStack(ManagerGenetics.java:133) ~[ManagerGenetics.class:?]
        at binnie.genetics.machine.analyser.Analyser.analyse(Analyser.java:37) ~[Analyser.class:?]
        at binnie.core.craftgui.WindowFieldKit.recieveGuiNBT(WindowFieldKit.java:277) ~[WindowFieldKit.class:?]
        at binnie.core.craftgui.minecraft.ContainerCraftGUI.recieveNBT(ContainerCraftGUI.java:470) ~[ContainerCraftGUI.class:?]
        at binnie.core.network.BinnieCorePacketID.onMessage(BinnieCorePacketID.java:50) ~[BinnieCorePacketID.class:?]
        at binnie.core.network.BinniePacketHandler.onMessage(BinniePacketHandler.java:26) ~[BinniePacketHandler.class:?]
        ... 20 more```

--- a/build.gradle
+++ b/build.gradle
@@ -22,7 +22,7 @@ buildscript {
 apply plugin: "se.bjurr.gitchangelog.git-changelog-gradle-plugin"
 apply plugin: 'forge'
 
-version = "2.0.18"
+version = "2.0.19"
 if (System.getenv("BUILD_NUMBER") != null) {
     version = version + "-" + System.getenv("BUILD_NUMBER")
 }

--- a/src/main/java/binnie/core/craftgui/WindowFieldKit.java
+++ b/src/main/java/binnie/core/craftgui/WindowFieldKit.java
@@ -274,7 +274,12 @@ public class WindowFieldKit extends Window {
 			return;
 		}
 
-		getWindowInventory().setInventorySlotContents(0, Analyser.analyse(getWindowInventory().getStackInSlot(0)));
+		ItemStack stack = getWindowInventory().getStackInSlot(0);
+		if (stack == null) {
+			return;
+		}
+
+		getWindowInventory().setInventorySlotContents(0, Analyser.analyse(stack));
 		getWindowInventory().decrStackSize(1, 1);
 	}
 

